cider/tests/functionality/CiderStringTest.cpp

```diff
@@ -26,7 +26,8 @@ class CiderStringTest : public CiderTestBase {
  public:
   CiderStringTest() {
     table_name_ = "test";
-    create_ddl_ = R"(CREATE TABLE test(col_1 INTEGER, col_2 VARCHAR(10));)";
+    create_ddl_ =
+        R"(CREATE TABLE test(col_1 INTEGER NOT NULL, col_2 VARCHAR(10) NOT NULL);)";
```

Comment:
What about nullable cases? in later PR?

Reply:
`convertToArrowRepresentation` didn't convert null buffer actually, will add null case when varchar data generator ready.